plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -210,15 +217,17 @@
           antenna_gain:
             description: |
               Specifies the antenna gain value in decibels (dB) for the 2.4GHz radio interface, valid values range
-              from 0 to 40. For example, 4.
+              from 0 to 40. For example, 10.
             type: int
             required: False
           radio_role_assignment:
             description: Role assignment mode for the 2.4GHz radio interface. Accepts "Auto", "Client-serving", or "Monitor". For example, Auto.
             type: str
             required: False
           cable_loss:
-            description: Cable loss in dB for the 2.4GHz radio interface. For example, 75.
+            description: |
+              Cable loss in dB for the 2.4GHz radio interface. Valid values range from 0 to 40.
+              This value should be less than the antenna gain value. For example: 2.
```

Comment:
Can you check this? You may get sanity error if you use :
For example: 2

Reply:
Updated